### ISSUE TITLE
Is OrchardCore 1.0 releasing soon?

### ISSUE THREAD

**Issue by randaratceridian** [OrchardCMS/OrchardCore#7167]
We are currently running RC 1. Upgrades for us are painful because we've made a lot of customizations. So we try to avoid them unless the cost/benefit is there. But we need Web Farm support (https://github.com/OrchardCMS/OrchardCore/issues/6303)

The Roadmap for 1.0 (https://github.com/OrchardCMS/OrchardCore/wiki/Roadmap) says September 2020. But originally 1.0 was scheduled for September 2019 (https://github.com/OrchardCMS/OrchardCore/wiki/Roadmap/a4277c874b4f7480c55f02d5a148e4bbb2ff0bcf).

So is 1.0 still scheduled for this month? There is only 1 day left. If no, when? We need to plan when we will upgrade.

**Comment by Skrypt** [comment]
I can't answer that with a date.

We've been pushing with few people contributing on OC in the last year. We all have also daytime jobs so it's hard for us to make any plan at this point. We all waiting on the Distributed Cache module, @jtkech demoed it last week to show that it works. And **_he's the only one working on it so far_**. I did not review the module, neither test it with my current production website in a staging environment to make sure everything works. At this point, not sure documentation is complete for this feature. Also, you can go to the issue tracker and click on milestones to see what remains to be fixed for shipping OC 1.0. We still have to do a major breaking change on the UserId's ... 

What can I say, if you want to have a distributed cache working right now I'd start using the ones in dev and see what's remaining to implement. I would not count on it to be feature complete as @deanmarcussen pointed out, the media needs to use also a distributed cache at some level. There might be some other adjustments needed to be done for this to work "as perfect as it can be".

I think OC 1.0 will be shipped when it's ready. Else, if you need a distributed cache right now ; help @jtkech to get it done or build one on top of your customized OC by yourself ... Don't make plans based on our delivery times which are not a priority for us to be met. Our priority is to ship a quality made software.

**Comment by sebastienros** [comment]
https://github.com/OrchardCMS/OrchardCore/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0

**Comment by agriffard** [comment]
See #7206
